pristine-zen-garden.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CSS Zen Garden: The Beauty of CSS Design</title>
+  <link rel="stylesheet" media="screen" href="/214/214.css?v=8may2013">
+  <link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Dave Shea">
+  <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+  <meta name="robots" content="all">
+</head>
+
+<body id="css-zen-garden">
+  <div class="page-wrapper">
+  <section class="intro" id="zen-intro">
```

Comment:
I think that the <section> tag needs to be indented from the <div> tag above it on line 15. Likewise the </aside> tag on line 176 would be indented from the </div> tag below it (this would close the "page-wrapper" class).